test/spec/proxySpec.js

```diff
@@ -1148,6 +1146,86 @@ describe('proxy', function() {
       // }, 100);
     });
   });
+  describe('Already proxified values', function() {
+    it("shouldn't re-proxify (it should deep clone) proxified values from different instances", function() {
```

Comment:
And a very kind reminder. This is 99.9% a waste of the-three-of-us time. I think it would make more sense to only worry about this if we actually face an issue because of it.

For this to be a concern you need:

1. Two JSONPatcherProxy instances, not two proxies. They both must be JSONPatcherProxy instances. Edge case.
2. You need to set the value of `JSONPatcherProxy_A.value` to `JSONPatcherProxy_B.value`. Edge case.
3.  `JSONPatcherProxy_B.value` has to be an object. A primitive value won't be an issue. Edge case.

The chance is virtually zero. Well more, but small enough not to invest time in it for the mean time.

I would be happy to warn in the console when you set an alien proxified object to a JSONPatcherProxy instance.